shared/ffmpeg/ffmpeg-version.ts

```diff
@@ -10,14 +13,10 @@ export function getFFmpegVersion () {
       return exec(`${ffmpegPath} -version`, (err, stdout) => {
         if (err) return rej(err)
 
-        const parsed = stdout.match(/ffmpeg version .?(\d+\.\d+(\.\d+)?)/)
-        if (!parsed?.[1]) return rej(new Error(`Could not find ffmpeg version in ${stdout}`))
+        const parsed = stdout.match(/(?<=ffmpeg version )[a-zA-Z\d.-]+/)
+        if (!parsed) return rej(new Error(`Could not find ffmpeg version in ${stdout}`))
 
-        // Fix ffmpeg version that does not include patch version (4.4 for example)
-        let version = parsed[1]
```

Comment:
Just a heads-up, I don't think this check was ever working properly.
The promise here wasn't being resolved, so the caller was probably just `await`ing forever.